tests/config_test.go

```diff
@@ -35,15 +36,18 @@ import (
 	"golang.org/x/crypto/ssh"
 
 	v1 "kubevirt.io/api/core/v1"
+	k8sv1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
 	"kubevirt.io/kubevirt/pkg/config"
 	"kubevirt.io/kubevirt/tests"
 	"kubevirt.io/kubevirt/tests/console"
 	"kubevirt.io/kubevirt/tests/decorators"
 	"kubevirt.io/kubevirt/tests/exec"
-	"kubevirt.io/kubevirt/tests/libpod"
+	"kubevirt.io/kubevirt/tests/framework/kubevirt"
 	"kubevirt.io/kubevirt/tests/libvmi"
 	"kubevirt.io/kubevirt/tests/testsuite"
+	"kubevirt.io/kubevirt/tests/libpod"
```

Comment:
Was this reordered by `make format`?

Reply:
@0xFelix  no sorry i did mistake while resloving merge conflict . shall i interchanged the /lipod and /kubevirt package.

Reply:
@0xFelix  Done